seaborn/rcmod.py

```diff
@@ -44,23 +39,21 @@
 
     ]
 
-if mpl_ge_2:
+_style_keys.extend([
```

Comment:
With the removal of the conditional, this code can be simplified further by just defining one list.

Reply:
Simplified, thanks!